tools/devtoolsNG/home.html

```diff
@@ -0,0 +1,41 @@
+<body>

```

Comment:
I see `//tools/` as mostly repo utilities. For example tools working with actual arcs stuff are in `//src/tools/`.

This file is a first tool being a client of the devtoolsNG protocol. I was planning that over time we would have multiple clients, e.g. Web-based GUI, command line REPL, specialized command line tools for particular debugging etc.

These tools should probably have very little dependency on the core Arcs code (like `//devtools/` does not depend on `//src/`), so that we can split them out into other repos or ensure that 3rd party devs can create their own (a bit hazy future, but you get the picture).

So in the medium terms I would expect something like:
```
//devtoolsNG/
       /gui/
       /repl/
```
maybe `gui` and `repl` are not best names, but you get the point.

Ideally at some point `devtoolsNg` will entirely replace the `//devtools/` and we will remove the `NG`, but it may take some time.

so...

WDYT about moving this to: `//devtoolsNG/basic/index.html` or something similar? index.html I think is good because HTTP servers will look at the index.html page if you don't give a file name (e.g. localhost://devtoolsNG/basic), but that's a tiny detail.